jobs/scanning/images-health/Jenkinsfile

```diff
@@ -58,5 +67,33 @@ node() {
             currentBuild.result = "FAILURE"
             throw exception  // gets us a stack trace FWIW
         }
+        
+        def week = [1:'Sunday', 2:'Monday', 3:'Tuesday', 4:'Wednesday', 5:'Thursday', 6:'Friday', 7:'Saturday']
+        Date today = new Date();
+        Calendar c = Calendar.getInstance();
+        c.setTime(today);
+        if ( week[c.get(Calendar.DAY_OF_WEEK)] == params.ALERT_DATE ) {  // to avoid spam message only check once on Mon
+            withCredentials([string(credentialsId: 'openshift-bot-token', variable: 'GITHUB_TOKEN')]) {
+                slackChannel = slacklib.to(BUILD_VERSION)
```

Comment:
we should have some kind of "summary" to #aos-art per the Jira, so it seems like these should be gathered for that purpose. however i'm really not sure what i'd want the summary to look like. a dump of hundreds of PRs is probably not it...